src/pages/Login/LoginKakao.js

```diff
@@ -0,0 +1,48 @@
+import { React } from 'react';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+
+const LoginKakao = () => {
+  const navigate = useNavigate();
+  const { Kakao } = window;
+  const login = () => {
+    Kakao.Kakao.init(process.env.REACT_APP_KAKAO_INIT_KEY);
```

Comment:
kakako를 window객체에서 구체분해 할당하고 있기 떄문에 Kakao.init으로 해주셔도 됩니다.
```suggestion
    Kakao.init(process.env.REACT_APP_KAKAO_INIT_KEY);
```

Reply:
넵 수정했습니다!